workflow/search-filter.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Search & Filter
+---
+
+The Search & Filter features serve as the jumping-off point in your data exploration process within Middleware. Data is made available across all contexts within Middleware, allowing you to understand the interconnectedness of your data.
+
+The Filter Bar and Search Fields are two of the simplest methods for sifting through your data.They are both dynamically generated based on the applications, integrations, and infrastructure you have instrumented within Middleware. These features are inclusive (i.e. union set operations) and will display data that satisfies any of the selected filters.
+
+<Note> For more information on log-based search, navigate to the [Log Monitoring - Search & Filter]() document to learn about our temporary log search solution.  </Note>
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExZXJmNDM5d2JmYzd4aTJ2OHg3ZHd3dms2bTJwZ3R1Z3dzcXl0anNkeiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/rEJAUGmdTiJZqvOEBT/giphy.gif" width="750" height="750"/>
+
+# Filtering by Date Range
+
+Filter your data by date and time with the Date Range picker in the top right screen of all pages within Middleware. Select your own date range from the calendar or choose from our default options:
+
+<img src="https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMWpuamp5amFsM3dtazFobXVibnhpb2xmc3NqOWN6d3djczM4anQydCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/9ZT90BmYhPrqB3fiWt/giphy.gif" width="750" height="750"/>
+
+# History & Live View
+
+Toggle between a static time range with the __History View__ or track your data in real-time with __Live View__. When you transition from the __History View__ to __Live View__, your page will display historical data no more than five minutes into the past by default.
+
+<Note> When you select a specific date range, you will be viewing your data in __History View__. Select the date range you previously specified to switch back to __Live View__.</Note>
```

Comment:
This is confusing me. Maybe the best way to say this is "Live Mode can be toggled on or off in the top right by clicking the History/Live button" + the video below which shows it p obviously.